spec/replace_browser_extension/02_user_interface_ios.rst

```diff
@@ -0,0 +1,197 @@
+---
+story: 60
```

Comment:
What means story 60 and spec 54?

Reply:
Ah it's the issue and PR number. How about linking them right in here?

Reply:
What happens if there is a spec update, i.e. there is another PR? Would we then add both numbers? I feel like we only need the reference to the main epic (#60) in that case. All PRs etc should be linked there, no?

Reply:
yes, true, only main epic should be there. I'll embedd the link